crates/modelardb_compression/src/lib.rs

```diff
@@ -233,7 +233,7 @@ fn can_models_be_merged(
 }
 
 /// A compressed segment being built from an uncompressed segment using the
-/// model types in [`models`](models). Each of the model types is used to
+/// model types in [`models`]. Each of the model types is used to
```

Comment:
As the lines in the middle of the doc comment has been shortened, the doc comment should be reformatted so each line fits as many words as possible without exceeding the maximum line width. Editors often has functionality to automate this task, e.g., Emacs provides the `fill-paragraph` command while Neovim and Vim provides similar functionality through the `gq` keybinding.

Reply:
Fixed.